docs/crates_io.md

```diff
@@ -0,0 +1,11 @@
+[![crates.io](https://img.shields.io/crates/v/cross.svg)](https://crates.io/crates/cross)
```

Comment:
this file should be renamed, it can even be ommited fully and just put in the lib.rs directly